x-pack/test/fleet_api_integration/apis/epm/install_by_upload.ts

```diff
@@ -68,15 +75,76 @@ export default function (providerContext: FtrProviderContext) {
       }
     });
 
-    it('should install a tar archive correctly', async function () {
+    async function uploadPackage() {
       const buf = fs.readFileSync(testPkgArchiveTgz);
-      const res = await supertest
+      return await supertest
         .post(`/api/fleet/epm/packages`)
         .set('kbn-xsrf', 'xxxx')
         .type('application/gzip')
         .send(buf)
         .expect(200);
+    }
+
+    it('should install a tar archive correctly', async function () {
+      const res = await uploadPackage();
+      expect(res.body.items.length).to.be(30);
+    });
+
+    it('should upgrade when uploading a newer tar archive', async () => {
```

Comment:
is it a tar or a zip file? the extension is .zip is that correct?

Reply:
yes, you're right